src/olympia/addons/indexers.py

```diff
@@ -170,7 +170,9 @@ def get_mapping(cls):
                         'strict_compatibility': {
                             'type': 'boolean', 'index': False},
                         'status': {'type': 'byte'},
-                        'webext_permissions_list': {
+                        'permissions': {
+                            'type': 'keyword', 'index': False},
+                        'optional_permissions': {
```

Comment:
@diox does that require a full reindex?

Reply:
Yes. It's also not backwards-compatible, which is a problem.

Reply:
I fixed this up in the serializer, as suggested by @diox on Matrix.